app/helpers/application_helper.rb

```diff
@@ -1,4 +1,11 @@
-# frozen_string_literal: true
-
-module ApplicationHelper
-end
+# frozen_string_literal: true
+
+module ApplicationHelper
+  include SessionsHelper
+
+  def follows?(follower_id)
+    return false unless logged_in?
+
+    Follower.where({ follower_user_id: current_user&.id, followed_user_id: follower_id }).any?
+  end
```

Comment:
Since application_helper is meant to be a lot of top-level stuff, consider making a followers_helper.rb and moving it there.